examples/chip-tool/commands/tests/TestCommand.h

```diff
@@ -73,7 +74,10 @@ class TestCommand : public TestRunner,
 
     CHIP_ERROR ContinueOnChipMainThread(CHIP_ERROR err) override;
 
-    chip::Controller::DeviceCommissioner & GetCurrentCommissioner() override { return CurrentCommissioner(); };
+    chip::Controller::DeviceCommissioner & GetCommissioner(const char * identity) override
+    {
+        return CHIPCommand::GetCommissioner(identity);
```

Comment:
This is because this method is defined by `src/app/tests/suites/commands/commissioner/CommissionerCommands.h` and needs to be `override`. But it happens that `CHIPCommand` does the right thing.